### PR TITLE
chore(weave): fix images damaged by exif stripping

### PR DESCRIPTION
**Pull request by andrewtruong**
Resolves some image issues introduced in wandb/weave#2863

### CODE REVIEW

File: docs/docs/guides/integrations/imgs/instructor/instructor_lm_trace.gif
Comment (andrewtruong) [inline docs/docs/guides/integrations/imgs/instructor/instructor_lm_trace.gif:1]: fixed

File: docs/docs/guides/integrations/imgs/litellm_gif.gif
Comment (andrewtruong) [inline docs/docs/guides/integrations/imgs/litellm_gif.gif:1]: fixed

File: docs/static/img/weave-hero.png
Comment (andrewtruong) [inline docs/static/img/weave-hero.png:1]: fixed

File: docs/static/img/docusaurus.png
Comment (andrewtruong) [inline docs/static/img/docusaurus.png:1]: fixed

**Review by jamie-rasmussen — APPROVED** [review]

**Pull request merged**